examples/tutorials/nb_python/ECCV_2020_Navigation.py

```diff
@@ -0,0 +1,1130 @@
+
+# %% [markdown]
+# # Full Config
+
+# %%
+# @title Configure Sim Settings
+
+test_scene = "./data/scene_datasets/mp3d/17DRP5sb8fy/17DRP5sb8fy.glb"
+
+rgb_sensor = True  # @param {type:"boolean"}
+depth_sensor = True  # @param {type:"boolean"}
+semantic_sensor = True  # @param {type:"boolean"}
+
+sim_settings = {
+    "width": 256,  # Spatial resolution of the observations
+    "height": 256,
+    "scene": test_scene,  # Scene path
+    "default_agent": 0,
+    "sensor_height": 1.5,  # Height of sensors in meters
+    "color_sensor": rgb_sensor,  # RGB sensor
+    "depth_sensor": depth_sensor,  # Depth sensor
+    "semantic_sensor": semantic_sensor,  # Semantic sensor
+    "seed": 1,  # used in the random navigation
+    "enable_physics": True,
+}
+
+
+# %%
+def make_cfg(settings):
+    sim_cfg = habitat_sim.SimulatorConfiguration()
+    sim_cfg.gpu_device_id = 0
+    sim_cfg.scene.id = settings["scene"]
+    sim_cfg.enable_physics = settings["enable_physics"]
+
+    # Note: all sensors must have the same resolution
+    sensors = {
+        "color_sensor": {
+            "sensor_type": habitat_sim.SensorType.COLOR,
+            "resolution": [settings["height"], settings["width"]],
+            "position": [0.0, settings["sensor_height"], 0.0],
+        },
+        "depth_sensor": {
+            "sensor_type": habitat_sim.SensorType.DEPTH,
+            "resolution": [settings["height"], settings["width"]],
+            "position": [0.0, settings["sensor_height"], 0.0],
+        },
+        "semantic_sensor": {
+            "sensor_type": habitat_sim.SensorType.SEMANTIC,
+            "resolution": [settings["height"], settings["width"]],
+            "position": [0.0, settings["sensor_height"], 0.0],
+        },
+    }
+
+    sensor_specs = []
+    for sensor_uuid, sensor_params in sensors.items():
+        if settings[sensor_uuid]:
+            sensor_spec = habitat_sim.SensorSpec()
+            sensor_spec.uuid = sensor_uuid
+            sensor_spec.sensor_type = sensor_params["sensor_type"]
+            sensor_spec.resolution = sensor_params["resolution"]
+            sensor_spec.position = sensor_params["position"]
+
+            sensor_specs.append(sensor_spec)
+
+    # Here you can specify the amount of displacement in a forward action and the turn angle
+    agent_cfg = habitat_sim.agent.AgentConfiguration()
+    agent_cfg.sensor_specifications = sensor_specs
+    agent_cfg.action_space = {
+        "move_forward": habitat_sim.agent.ActionSpec(
+            "move_forward", habitat_sim.agent.ActuationSpec(amount=0.25)
+        ),
+        "turn_left": habitat_sim.agent.ActionSpec(
+            "turn_left", habitat_sim.agent.ActuationSpec(amount=30.0)
+        ),
+        "turn_right": habitat_sim.agent.ActionSpec(
+            "turn_right", habitat_sim.agent.ActuationSpec(amount=30.0)
+        ),
+    }
+
+    return habitat_sim.Configuration(sim_cfg, [agent_cfg])
+
+
+# %%
+cfg = make_cfg(sim_settings)
```

Comment:
You can wrap this stuff in a if __name__ == `__main__` if you want, but it's not required.